src/or/onion.h

```diff
@@ -121,5 +160,50 @@ int extend_cell_format(uint8_t *command_out, uint16_t *len_out,
 int extended_cell_format(uint8_t *command_out, uint16_t *len_out,
                          uint8_t *payload_out, const extended_cell_t *cell_in);
 
+// TODO: Remove these defines when we implement the first handshake that uses
+// CREATE(D)2V cells. --isis
+#ifndef ONION_HANDSHAKE_TYPE_NTOR2_NULL
+#define ONION_HANDSHAKE_TYPE_NTOR2_NULL 0x0003
```

Comment:
Let's not use 3 even as a temporary value; instead let's reserve some range for temporary use